config/1.0.0/sample_config_1.0.0.json

```diff
@@ -18,96 +18,151 @@
     "SPL[1234].+": "SMAP",
     "VIIRS_NPP-.+-L2P": "VIIRS_PO"
   },
-  "Excluded_Science_Variables": [
+  "ExcludedScienceVariables": [
     {
       "Applicability": {
         "Mission": "ICESat2"
       },
-      "Variable_Pattern": [
+      "VariablePattern": [
         "/quality_assessment/.*",
         "/orbit_info/.*",
         "/atlas_impulse_response/.*"
       ]
     }
   ],
-  "Required_Fields": [
+  "RequiredVariables": [
     {
       "Applicability": {
         "Mission": "GEDI"
       },
-      "Variable_Pattern": [
+      "VariablePattern": [
         ".*shot_number"
       ]
     }
   ],
-  "ProductEpochs": [
+  "CFOverrides": [
```

Comment:
do it.  `MetadataOverrides`

Reply:
[578faf8](578faf8974eb086f0f60841a47d70f1ae47a008e) :tada: